site/_posts/2017-04-17-Announcing-Euwe-3.html.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Euwe-3
+author: jprause
+date: 2017-04-17 07:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+
+We've just built Euwe-3. This release contains a security fix, bug fixes, numerous UI tweaks, and stabilization.
+You can download the Euwe-3 release [here](http://manageiq.org/download/)
+
+
+Security Fixes:
+
+[CVE-2017-2653](https://access.redhat.com/security/cve/CVE-2017-2653) - A number of unused delete routes are present in CloudForms which can be accessed via GET requests instead of just POST requests. This could allow an attacker to bypass the protect_from_forgery XSRF protection causing the routes to be used. This attack would require additional cross-site scripting or similar attacks in order to execute.
+This issue was reported by Martin Povolny (Red Hat).
+
+and here are just a few of the things added since Euwe-2 release
+- UI: Add new Subnet must be disabled when there is no cloud provider present
+- Inventory refresh fail on timeout, after ~2 minutes
+- Security groups/Floating IPs not displayed in Network Topology View
+- External authentication works when logging into the Admin UI but doesn't work for the same user to get into the Service UI
+- Unable to bring VM out of retirement from details page 
```

Comment:
@JPrause I was wondering where these PRs are from and couldn't find matching ones in the changelog. Then I notice they are titles of CFME bugs. Even though we maintain close downstream/upstream development, let's not mix them for clarity's sake. Perhaps you can use the description in the corresponding ManageIQ PRs/commits, or even directly from the MIQ changelog.